tempesta_fw/http_parser.c

```diff
@@ -1351,6 +1353,7 @@ __req_parse_cache_control(TfwHttpReq *req, unsigned char *data, size_t len)
 	__FSM_STATE(Req_I_CC_m) {
 		TRY_STR("max-age=", Req_I_CC_MaxAgeV);
 		TRY_STR("min-fresh=", Req_I_CC_MinFreshV);
+		TRY_STR("max_stale", Req_I_CC_MaxStale);
```

Comment:
It seems the field is non-standard-defined, at least comment when we can see this is required.